packages/webapp/pages/_app.tsx

```diff
@@ -39,6 +39,7 @@ import { useLazyModal } from '@dailydotdev/shared/src/hooks/useLazyModal';
 import { usePrompt } from '@dailydotdev/shared/src/hooks/usePrompt';
 import { ReactQueryDevtools } from 'react-query/devtools';
 import { defaultQueryClientConfig } from '@dailydotdev/shared/src/lib/query';
+import { EXTENSION_PERMISSION_KEY } from '@dailydotdev/shared/src/hooks/useExtensionPermission';
```

Comment:
Assume we can remove these right, as it doesn't see use (might have fixed it with build lints already)